src/components/scenes/SendConfirmationScene.js

```diff
@@ -341,34 +337,21 @@ export class SendConfirmation extends Component<Props, State> {
   }
 
   onExchangeAmountChanged = async ({ nativeAmount, exchangeAmount }: ExchangedFlipInputAmounts) => {
-    const { spendingLimits, fiatPerCrypto, coreWallet, sceneState, currencyCode, newSpendInfo, updateTransaction } = this.props
-    this.setState({
-      showSpinner: true
-    })
-    const count = ++this.count
+    const { fiatPerCrypto, coreWallet, sceneState, currencyCode, newSpendInfo, updateTransaction, getAuthRequiredDispatch } = this.props
+    this.setState({ showSpinner: true })
     const amountFiatString: string = bns.mul(exchangeAmount, fiatPerCrypto.toString())
     const amountFiat: number = parseFloat(amountFiatString)
     const metadata: EdgeMetadata = { amountFiat }
     const guiMakeSpendInfo = { nativeAmount, metadata }
 
     const guiMakeSpendInfoClone = { ...guiMakeSpendInfo }
     const spendInfo = getSpendInfoWithoutState(guiMakeSpendInfoClone, sceneState, currencyCode)
-    let authType
-    if (spendingLimits.transaction.isEnabled) {
-      authType = amountFiat > spendingLimits.transaction.amount ? 'pin' : 'none'
-    } else {
-      authType = 'none'
-    }
+    const authType: any = getAuthRequiredDispatch(spendInfo) // Type casting any cause dispatch returns a function
```

Comment:
This should not need to be type casted. Dispatch does not return to function but actually returns a value

Reply:
Lot of errors on flow as it is use on newSpendInfo(spendInfo, authType) which the second parameter expects a type